rct229/utils/std_comparisons.py

```diff
@@ -26,6 +31,80 @@ def std_equal(
     Returns
     -------
     bool
-        True iff the val is within percent_tolerance of std_val
+        True if the val is within percent_tolerance of std_val
     """
     return abs(std_val - val) <= (percent_tolerance / 100) * abs(std_val)
+
+
+def std_equal_with_precision(
+    val: "Quantity", std_val: "Quantity", precision: "Quantity"
```

Comment:
Instead of "Quantity" (str), it should be Quantity.

Reply:
woops! thanks